bitcoin/src/Bitcoin/Keys/Common.hs

```diff
@@ -44,17 +44,7 @@ import Control.DeepSeq (NFData)
 import Control.Monad (guard, mzero, (<=<))
 import Crypto.Hash (hashWith)
 import Crypto.Hash.Algorithms (SHA256 (SHA256))
-import Crypto.Secp256k1 (
-    PubKey,
-    SecKey (..),
-    derivePubKey,
-    exportPubKey,
-    importPubKey,
-    secKey,
-    tweak,
-    tweakAddPubKey,
-    tweakAddSecKey,
- )
+import Crypto.Secp256k1 (PubKeyXY, SecKey, derivePubKey, exportPubKeyXY, exportSecKey, importPubKeyXY, importSecKey, importTweak, pubKeyTweakAdd, secKeyTweakAdd)
```

Comment:
nit: I prefer the vertical layout. Is there a reason to have these long lines like this?

Reply:
I think that was a choice that fourmolu made. I'll see if I can nudge it to a vertical layout. I agree that vertical is better but the thing I care most about is that this is handled automatically by the formatter.